web/gui/dashboard_info.js

```diff
@@ -127,6 +127,10 @@ netdataDashboard.menu = {
         info: 'Charts with performance information for all the system disks. Special care has been given to present disk performance metrics in a way compatible with <code>iostat -x</code>. netdata by default prevents rendering performance charts for individual partitions and unmounted virtual disks. Disabled charts can still be enabled by configuring the relative settings in the netdata configuration file.'
     },
 
+    'mdstat': {
+        icon: '<i class="fas fa-hdd"></i>',
```

Comment:
Do we want it to keep appearing as `mdstat`? `mdstat` is a file, usually we don't name section like that 🤷‍♂️ 

Could be:
 - `MD devices`
 - `Software RAID`

Reply:
I will add this comment for the main issue, and when I touch the families in the next PRs, I will rename it.

Reply:
What families? That is just a question. What do you think @vlvkobal?

Reply:
We will add the family `Filesystems` to our submenu, this was the motive I said that when I work with Families, I could bring this. 
I agree 100% that `mdstat` is not the best name.

Reply:
I would use `MD arrays` or `Linux RAID`. `devices` and `Software RAID` sound too general.

Reply:
That is what it is. I got it from `man mdadm`.

> mdadm - manage MD devices aka Linux Software RAID.

Reply:
Anyway, we are in agreement that we need something better then `mdstat` 😄 👍

Reply:
All right, now we have `MD arrays`, please review it again and thank you for your contribution :smile: !